course3/homework-string/test.cc

```diff
@@ -1,3 +1,4 @@
+#include "stdafx.h"
```

Comment:
I wouldn't put this into the code, this is some VS stuff. I would try to make it a standard C++